kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/cloud/Cloudlet.java

```diff
@@ -79,7 +78,7 @@ public void unsetCloudService(CloudService cloudService) {
     //
     // ----------------------------------------------------------------
 
-    protected void activate(ComponentContext componentContext) {
```

Comment:
As we are using Java 8 now, we could be providing a default interface method forwarding calls to the old method. That way we would not break the interface.